Dockerfile

```
@@ -0,0 +1,5 @@
+FROM rust:1.75
+WORKDIR /app
+COPY . .
+RUN cargo install --path .
+CMD ["mineginx"]
```

Hello, thank you for your pull request!

Can I ask some questions?  
So, what's about use `scratch` as base image for out mineginx container?

Due to this, the result container will become very small

```suggestion
FROM rust:1.75 as build
WORKDIR /src
COPY . .
RUN rustup target add x86_64-unknown-linux-musl
RUN cargo b -r --target=x86_64-unknown-linux-musl

FROM scratch
WORKDIR /app
COPY --from=build ./src/target/x86_64-unknown-linux-musl/release/mineginx .
COPY --from=build ./src/config/ ./config/
CMD [ "./mineginx" ]
```

That is my builded containers
first is based on `rust:1.75`
second is based on `scratch`

```
folleach:~# docker image ls
second            latest    a3e0db460954   9 minutes ago       5.58MB
first             latest    c6918457f70b   About an hour ago   2.87GB
``` 

In addition, I can do pipeline for image publishing to the DockerHub, if you wish

---

Let me try this and use it in my deployment real quick

---

Thanks for the clarification, I'm not very good at docker.  
It seems to me that there is no difference in maintenance between an installed as a global binary in a container or a copied statically-linked binary.

Of course, I don't even have bash in my container, but I don't think need it.  

Write as you try this, if you have any more fixes that have been found in your environment - welcome